src/components/screens/DocsScreen/VersionSelector.tsx

```diff
@@ -1,113 +1,59 @@
+const stylizeVersion = ({ label, string }: { label?: string; string: string }) =>
+  label ? `${string} (${label})` : string;
 
-const Wrapper = styled.div`
-  font-size: ${typography.size.s2}px;
-`;
-
-const Version = styled.span`
-  color: ${color.darker};
-  display: none;
-  @media (min-width: ${breakpoint * 1}px) {
-    display: inline;
-  }
-`;
-
-const VersionLink = styled(Link)`
-  svg {
-    transform: rotate(90deg);
-  }
-`;
+interface Stable {
+  version: number;
+  string: string;
+  label?: 'latest';
+}
 
-const LinkHeading = styled(Subheading)`
-  display: block;
-  font-size: 10px;
-  font-weight: 800;
-  letter-spacing: 4.29px;
-  line-height: 18px;
-  color: ${color.mediumdark};
-  padding: 7px 15px;
-  border-bottom: 1px solid ${color.border};
-  ${(props) => props.withTopBorder && `border-top: 1px solid ${color.border};`}
-`;
+interface PreRelease {
+  version: number;
+  string: string;
+  label: 'alpha' | 'beta' | 'rc';
+}
 
-const VersionLinkList = styled(TooltipLinkList)`
-  border-radius: 0;
-`;
+export interface Versions {
+  stable: Stable[];
+  preRelease: PreRelease[];
+}
 
-const stylizeVersion = ({ label, string }) => (label ? `${string} (${label})` : string);
+interface VersionSelectorProps {
+  framework: string;
+  version: number;
+  slug: string;
+  versions: Versions;
+}
 
-export function VersionSelector({ framework, version, versions, slug, tooltipProps, ...rest }) {
-  const getVersionLink = ({ label, string }) => ({
-    LinkWrapper: GatsbyLinkWrapper,
-    href: buildPathWithFramework(slug, framework, string),
-    title: stylizeVersion({ label, string }),
+export function VersionSelector({ framework, version, versions, slug }: VersionSelectorProps) {
+  const getVersionLink = ({ label, string }: { label?: string; string: string }) => ({
+    label: stylizeVersion({ label, string }),
+    link: { url: buildPathWithFramework(slug, framework, string), LinkWrapper: GatsbyLinkWrapper },
   });
 
   const stableLinks = versions.stable.map(getVersionLink);
   const preReleaseLinks = versions.preRelease.map(getVersionLink);
 
-  return (
-    <Wrapper {...rest}>
-      <Version>Version: </Version>
-      <WithTooltip
-        placement="bottom"
-        trigger="click"
-        tooltip={
-          <>
-            <LinkHeading>Stable</LinkHeading>
-            <VersionLinkList links={stableLinks} />
-            {preReleaseLinks.length > 0 && (
-              <>
-                <LinkHeading withTopBorder>Pre-release</LinkHeading>
-                <VersionLinkList links={preReleaseLinks} />
-              </>
-            )}
-          </>
-        }
-        as="span"
-        {...tooltipProps}
-      >
-        <VersionLink isButton primary withArrow>
-          {stylizeVersion(
-            [...versions.stable, ...versions.preRelease].find(({ version: v }) => v === version)
-          )}
-        </VersionLink>
-      </WithTooltip>
-    </Wrapper>
+  const versionOptions = [
+    {
+      label: 'stable',
```

Comment:
I think these `label`s should be capitalized.

Reply:
Like 'Stable' instead of `stable`?

Reply:
Yep. Same for "Pre-release".

Reply:
Ha. I left these comments before looking at the snapshots/Storybook. I see now that the visual treatment renders this moot. 🤦‍♂️ Sorry for the noise!

I'll take the opportunity to say that both FrameworkSelector and VersionSelector seem sound and look great to me!